Health-Application-Saturday/.gitignore

```diff
@@ -35,3 +35,6 @@ out/
 
 ### VS Code ###
 .vscode/
```

Comment:
Almost there. 
You want to add .idea the git ignore. It's still showing up on

Reply:
@kaixin2017 Add in the .idea to the ignore and remove the .idea from the remote and you should be good to go on merging